GAME/DELTAPAK.C

```diff
@@ -2120,9 +2120,29 @@ void GetActorJointAbsPosition(int actornum, unsigned long nodenum, struct PHD_VE
 	S_Warn("[GetActorJointAbsPosition] - Unimplemented!\n");
 }
 
+void GrabActorMatrix(int actornum, int nodenum, struct MATRIX3D* matrix)
+{
+	S_Warn("[GrabActorMatrix] - Unimplemented!\n");
+}
+
 void deal_with_actor_shooting(unsigned short* shootdata, int actornum, int nodenum, struct PHD_VECTOR* pos)
 {
-	S_Warn("[deal_with_actor_shooting] - Unimplemented!\n");
+	for(int i = 0; shootdata[i] != -1; i++)
```

Comment:
All variables must be declared at the start of a method.

Reply:
C89 states that variables must be declared at the start of a scope block, which is the case here